WordPress/gutenberg
# Query No Results: Add Border & Spacing Support

## What?

Add border and Spacing support to the `Query No Results` block.

Part of https://github.com/WordPress/gutenberg/issues/43247

## Why?
`Query No Results` block is missing border and Spacing support.

## How?

Add the border and Spacing support in block.json.

## Testing Instructions

- Go to Global Styles Settings. ( Under Appearance > Editor > Styles > Edit styles > Blocks )
- Make sure that `Query No Results` block's border and Spacing is Configurable via Global Styles.
- Edit Archives Template &  Add `Query No Results` block and Apply the border Styles and Spacing.
- Verify that `Query No Results` block styles take precedence over global Styles.
- Verify that `Query No Results` block borders and Spacing display correctly in both the Editor and Frontend.

## Screenshots or Screencast 


https://github.com/user-attachments/assets/3c801237-0724-4d6b-b81a-e9b2c3c36a62
